core/src/main/resources/application.properties

```diff
@@ -98,7 +98,7 @@ klaw.uiapi.servers=https://localhost:9097
 # Make sure node, pnpm are installed.
 # If the above are already installed, mvn install will build and copy the coral assets for you.
 # Alternatively you can go through https://github.com/aiven/klaw/blob/main/coral/README.md
-klaw.coral.enabled=false
+klaw.coral.enabled=true
 
```

Comment:
This is one should we leave this as false or should we have it as true??

Reply:
It should be false, I just added it to my commit wrongly 😬